front/app/component-library/components/Title/index.tsx

```diff
@@ -23,7 +23,7 @@ import Box, {
 
 export type Variant = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
 export type FontSize = keyof typeof fontSizes;
-export type FontWeight = 'bold' | 'normal';
+export type FontWeight = 'bold' | 'normal' | 600;
 type FontStyle = 'italic' | 'normal';
```

Comment:
I'm not sure why we use different types for Title vs Text, but it would also be higher effort to use just 1 version of all these types, so I'm going to leave it as-is for now :thinking: